File: SemantikosClientesWS/ClienteJava/src/cl/minsal/semantikos/ws/shared/CrossmapSet.java

package cl.minsal.semantikos.ws.shared;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlType;


/**
 * <p>Java class for CrossmapSet complex type.
 * 
 * <p>The following schema fragment specifies the expected content contained within this class.
 * 
 * <pre>
 * &lt;complexType name="CrossmapSet">
 *   &lt;complexContent>
 *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
 *       &lt;sequence>
 *         &lt;element name="abbreviatedName" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="name" type="{http://www.w3.org/2001/XMLSchema}string" minOccurs="0"/>
 *         &lt;element name="version" type="{http://www.w3.org/2001/XMLSchema}int"/>
 *       &lt;/sequence>
 *     &lt;/restriction>
 *   &lt;/complexContent>
 * &lt;/complexType>
 * </pre>
 * 
 * 
 */
@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "CrossmapSet", propOrder = {
    "abbreviatedName",
    "name",
    "version"
})
public class CrossmapSet {

    protected String abbreviatedName;
    protected String name;
    protected int version;

    /**
     * Gets the value of the abbreviatedName property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getAbbreviatedName() {
        return abbreviatedName;
    }

    /**
     * Sets the value of the abbreviatedName property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setAbbreviatedName(String value) {
        this.abbreviatedName = value;
    }

    /**
     * Gets the value of the name property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getName() {
        return name;
    }

    /**
     * Sets the value of the name property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setName(String value) {
        this.name = value;
    }

    /**
     * Gets the value of the version property.
     * 
     */
    public int getVersion() {
        return version;
    }

    /**
     * Sets the value of the version property.
     * 
     */
    public void setVersion(int value) {
        this.version = value;
    }

}

File: SemantikosClientesWS/clientePHP/src/Semantikos/ClientBundle/API/CrossmapSet.php
<?php

namespace Semantikos\ClientBundle\API;

class CrossmapSet
{

    /**
     * @var string $abbreviatedName
     */
    protected $abbreviatedName = null;

    /**
     * @var string $name
     */
    protected $name = null;

    /**
     * @var int $version
     */
    protected $version = null;

    /**
     * @param int $version
     */
    public function __construct($version)
    {
      $this->version = $version;
    }

    /**
     * @return string
     */
    public function getAbbreviatedName()
    {
      return $this->abbreviatedName;
    }

    /**
     * @param string $abbreviatedName
     * @return CrossmapSet
     */
    public function setAbbreviatedName($abbreviatedName)
    {
      $this->abbreviatedName = $abbreviatedName;
      return $this;
    }

    /**
     * @return string
     */
    public function getName()
    {
      return $this->name;
    }

    /**
     * @param string $name
     * @return CrossmapSet
     */
    public function setName($name)
    {
      $this->name = $name;
      return $this;
    }

    /**
     * @return int
     */
    public function getVersion()
    {
      return $this->version;
    }

    /**
     * @param int $version
     * @return CrossmapSet
     */
    public function setVersion($version)
    {
      $this->version = $version;
      return $this;
    }

}
